lib/webauthn/authenticator_assertion_response.rb

```diff
@@ -14,7 +14,7 @@ def self.from_client(response, relying_party: WebAuthn.configuration.relying_par
       encoder = relying_party.encoder
 
       user_handle =
-        if response["userHandle"] && String === response["userHandle"]
+        if response["userHandle"] && response["userHandle"].is_a? String
```

Comment:
Seems that we have a syntax error here 😕 

```suggestion
if response["userHandle"] && response["userHandle"].is_a?(String)
```